packages/smoldot-provider/src/SmoldotProvider.ts

```diff
@@ -34,7 +37,7 @@ interface SubscriptionHandler {
 
 interface StateSubscription extends SubscriptionHandler {
     method: string;
-    params: any[];
+    params: any[]; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
You should not disable linter. Any should not be used

Reply:
This came from PolkadotJS ... I wanted to keep it the same as the original code intentionally